.github/DEVELOPER.md

```diff
@@ -7,6 +7,13 @@
 * Your favorite IDE
 * kubebuilder
 
+### Install Kubebuilder
```

Comment:
why are you moving this around

Reply:
Accidental commit.. i was separately working on tweaking the docs but then I gave up.